com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -1,5 +1,164 @@
+- [3e96cf95] (2021-06-08) Jean-Sébastien Fauteux / feat: Add RPC Name Lookup Table Provided by NetworkBehaviourILPP (#875)
+- [3495445e] (2021-06-03) Jesse Olmer / fix: update package version to 0.2.0 because of unity minversion change (#881)
+- [0444c039] (2021-06-02) Jesse Olmer / fix: Update package patch version to allow package registry re-publish
+- [fa15fc6f] (2021-06-01) Jesse Olmer / docs: Fix typo in changelog version title
+- [bd223cfb] (2021-06-01) Lori Krell / docs: Hotfix Changelog for 0.1.1 and manual update (#873)
+- [39b56366] (2021-06-01) Jesse Olmer / fix: Update package patch version to allow package registry re-publish
+- [4b15869f] (2021-05-21) Sam Bellomo / fix: Adding exception for silent failure for clients getting other player's object #844Merge pull request #844 from Unity-Technologies/feature/adding-exception-for-client-side-player-object-get
+- [63436440] (2021-05-21) Samuel Bellomo / Merge branch 'develop' into feature/adding-exception-for-client-side-player-object-get
+- [7561c341] (2021-05-21) Samuel Bellomo / adding null check and spacing fix
+- [e2b17b10] (2021-05-21) Samuel Bellomo / some cleanup
+- [3566ea04] (2021-05-20) Samuel Bellomo / fixing a few issues when connecting and disconnecting additional clients Adding separate tests in SpawnManagerTests. Added Teardown
+- [b3c155b5] (2021-05-20) Samuel Bellomo / Merge branch 'develop' into feature/adding-exception-for-client-side-player-object-get
+- [d783a4e0] (2021-05-20) Samuel Bellomo / adding more tests
+- [e2fd839c] (2021-05-19) Samuel Bellomo / Adding tests for that exception Adding the possibility to have multiple clients in MultiInstanceHelpers Updating exception check to make sure to use local networkmanager (so it works with tests)
+- [d11e22be] (2021-05-19) Sam Bellomo / feat: NetworkTransform now uses NetworkVariables instead of RPCs (#826)
+- [ad8ae404] (2021-05-18) Samuel Bellomo / Adding proper exception for invalid case. This is so users don't have silent failures calling this client side expecting to see other player objects. This solves issue https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/issues/581
+- [e409b373] (2021-05-18) Andrew Spiering / fix: Fixing utp license file for legal (#843)
+- [11b2d250] (2021-05-10) Luke Stampfli / refactor: reduce dictionary lookups (#584)
+- [e864e8eb] (2021-05-03) Phil Deschain / feat: OnAllClientsReady (#755)
+- [b7357f85] (2021-04-19) Andrew Spiering / ci: Enabling MLAPI UTP Transport package (#736)
+- [c0386eb3] (2021-04-15) Jesse Olmer / Merge branch 'master' into develop
+- [68482608] (2021-04-15) Cosmin / docs: Remove MIT license hyperlink in the main README.md (#732)
+- [644c71a1] (2021-04-14) Jesse Olmer / Merge branch 'master' into develop
+- [0f7d388b] (2021-04-14) Jesse Olmer / Merge tag '0.1.0' into develop
+- [6984f6d6] (2021-04-14) will-mearns / docs: add "expected outcome" section to bug report template (#728)
```

Comment:
This doesn't have to be in the changelog :p

Reply:
the list is just a `git log` dump :)